exasol_udf_mock_python/mock_context_run_wrapper.py

```diff
@@ -24,6 +27,8 @@ def __init__(self, mock_context: MockContext, input_type: str, output_type: str)
             self.get_dataframe = self._mock_context.get_dataframe
             self.size = self._mock_context.size
 
-
     def __getattr__(self, name):
         return self._mock_context.__getattr__(name)
+
+    def __getitem__(self, item):
+        return self._mock_context._data[item]
```

Comment:
@umitbuyuksahin I am not sure, if this is correct for all cases, because I think, getitem is only available in case you have a variadic udf and in that case, you don't have getattr. Also, the columns names for variadic udfs are only number, in the current implementation they can be anything. A proper solution would have been to introduce an option to the MockMetadata that you want to mock a variadic udf. Please, add an issue to this project to fix this.

Reply:
A ticket is created for that:
https://github.com/exasol/udf-mock-python/issues/30